### PR TITLE
Added more reporters for single request

### PR DESCRIPTION
**Pull request by raghavbhatia336**

### CODE REVIEW

File: test/cli/request/request-options.test.js [hunk 1]
```diff
@@ -84,3 +84,29 @@ describe('Curl Options', function () {
...
+        // eslint-disable-next-line max-len
+        exec('node ./bin/newman.js request -X GET "https://5a178277-8664-409c-8e93-d5ee2935d2cb.mock.pstmn.io/get-json" -r cli', function (code) {
+            expect(code, 'should have exit code of 0').to.equal(0);
+            done();
+        });
+    });
+
+    it('should correctly work with progress reporter options', function (done) {
+        // eslint-disable-next-line max-len, quotes
+        exec('node ./bin/newman.js request -X GET "https://5a178277-8664-409c-8e93-d5ee2935d2cb.mock.pstmn.io/get-json" -r progress"', function (code) {
```
Comment (raghavgarg1257) [inline test/cli/request/request-options.test.js]: Extra `"`, is present at the end in the `exec` command, which is why tests are failing.

**Comment by raghavgarg1257** [comment]
@raghavbhatia336 Let's merge this PR and add error handling for reporter initialization in the next phase.

**Comment by shamasis** [comment]
Just one side comment. We are inflating this PR to have a lot of tiny things to sneak in. For example, this is a PR for single request and I'm seeing fixes for adding teamcity help text, fixes for CLI timing phases, etc. 😊

They might get in the way for eventual merge approval into Newman.

**Comment by raghavgarg1257** [comment]
Hello @shamasis, Thanks for the comment, the changes in this PR are all related to single-request, but I realize I should have created a separate PR for the small fixes and not clutter the feature of enabling the default reporters for single-request. 👍 

The files `lib/run/index.js` and `lib/request/index.js` are very similar in terms of initiation of `postman-runtime` and using the same default reporters, for which we have copied the config for now and will be DRYing it based on modularity we would want to keep between them. 😅

**Pull request merged**